src/Microsoft.AspNet.Http.Features/FeatureReference.cs

```diff
@@ -1,38 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-namespace Microsoft.AspNet.Http.Features
```

Comment:
@benaadams Bring this type back then I'll be ready to merge this PR. Also squash the commits.